src/org/redalert1741/steamworks/Shooter.java

```diff
@@ -35,6 +35,14 @@ public Shooter(CANTalon m)
 	
 	public void setSpeed(double rpm)
 	{
+		if(rpm == 0)
+		{
+			flyWheel.changeControlMode(TalonControlMode.PercentVbus);
+		}
+		else
+		{
+			flyWheel.changeControlMode(TalonControlMode.Speed);
+		}

```

Comment:
Just curious as to why it's like this?  Is there an issue with setting it to `0` in speed control mode?